rca/events/models.py

```diff
@@ -26,14 +27,132 @@
 )
 
 from .blocks import CallToAction, EventDetailPageBlock, PartnersBlock
-from .forms import EventPageAdminForm
+from .forms import EventIndexPageAdminForm, EventPageAdminForm
 
 
-class EventIndexPage(BasePage):
+class EventIndexPageRelatedEditorialPage(Orderable):
+    page = models.ForeignKey(
+        "events.EventDetailPage",
+        null=True,
+        blank=False,
```

Comment:
Shouldn't this be required and not nullable?

Reply:
Yep - thanks